src/main/java/bio/terra/service/dataset/flight/ingest/SkippableStep.java

```diff
@@ -0,0 +1,49 @@
+package bio.terra.service.dataset.flight.ingest;
+
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import java.util.function.Predicate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class SkippableStep implements Step {
+
+  private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Reply:
Needs to not be static so that the implementing class name is available.

Reply:
I don't think this is the usual practice in our code, even if a class is subclassed. Instead we always use the declared class that the log statement is inside, not the class type at runtime.